script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1046,17 +1217,17 @@ In other words―――
 [sha:a233471bd01a8f702036656f61e540d26e93263f]{
 -- Page 227, Offset 22716.
 -- それは、何があっても死なないという事。
-She wouldn't die, no matter what.
+It would be absolutely impossible for her to die.
 }
 [sha:1b9023982c212b4036bbb255bafef8085b34bc8c]{
 -- Page 227, Offset 22717.
 -- 死するが運命だからこそ転生を求めた男とは、真逆の在り方に他ならない。
-It's the exact opposite of the man who sought reincarnation precisely because he was fated to die.
+Her mode of existence is the polar opposite of the man who chased immortality because he was destined to die.
```

Comment:
arikata as 'mode of existence' feels unnecessarily literal. 
Also, no need to change reincarnation to immortality.
Nobody is gonna be confused.
>The polar opposite existence of the man who sought reincarnation %{i}because%{/i} death was his fate.